pages/docs/tutorials/getting-started/slackbot-websocket-tutorial.md

```diff
@@ -0,0 +1,340 @@
+---
+title: Create an AsyncAPI Document for a SlackBot with WebSockets
+description: In this tutorial, you'll learn how to create an AsyncAPI document designed for a Slack application that operates in Socket Mode using the WebSockets protocol.
+weight: 80
+---
+
+## Introduction
+In this tutorial, you will learn how to generate an AsyncAPI document designed for a Slack application that operates in Socket Mode. The aim is to help you grasp a real-world application of AsyncAPI with the WebSocket protocol.
+
+Consider a scenario where you are in charge of maintaining a highly active Slack workspace. You want an easy way to keep track of the popular messages across all the channels but doing this manually would be a difficult task. To simplify this process, you’re going to build a Slackbot called `Heart-Counter-Slack-Bot` that actively monitors reactions added to a message and determine its popularity by counting the reactions of the “heart” emoji.
+
+Here’s a visual representation of how `Heart-Counter-Slack-Bot` should work: 
+
+```mermaid
+sequenceDiagram
+participant Heart-Counter-Slack-Bot
+participant Slack Server (Socket Mode)
+participant User
+
+
+Heart-Counter-Slack-Bot ->>Slack Server (Socket Mode): Connect
+Note right of Heart-Counter-Slack-Bot: Establish WebSocket connection
+
+
+Slack Server (Socket Mode)->> Heart-Counter-Slack-Bot : Handshake
+Note left of Slack Server (Socket Mode): Establish Socket Mode connection
+
+Slack Server (Socket Mode)->> Heart-Counter-Slack-Bot : Sends "hello" event
+Note left of Slack Server (Socket Mode): Confirms successful connection
+
+
+
+
+
+User-->>Slack Server (Socket Mode): Adds emoji to a message.
+
+
+Slack Server (Socket Mode)->>Heart-Counter-Slack-Bot: Sending "reaction_added" payload
+
+
+Note left of Heart-Counter-Slack-Bot: Event received
+``` 
+
+## Background context
+[WebSocket](https://en.wikipedia.org/wiki/WebSocket) is a communication protocol that enables simultaneous bidirectional data exchange between a client and a server over a single, long-lived connection. Unlike HTTP, which relies on the request-response model, WebSocket is ideal for scenarios where real-time, interactive and low-latency communication is necessary.
+
+
+```mermaid
+sequenceDiagram
+Client -->> Server: Initiate WebSocket Connection
+Server -->> Client: WebSocket Response
+Server -->> Client: WebSocket Response
+Server -->> Client: WebSocket Response
+Server -->> Client: WebSocket Response
+```
+
+In Slack, WebSocket is employed as part of its [Socket Mode](https://api.slack.com/apis/connections/socket) feature to facilitate real-time notifications between Slack's servers and third-party applications or bots. The [Slack Event API](https://api.slack.com/apis/connections/events-api) is a tool that lets you receive real-time notifications of specific events in a Slack workspace such as messages, reactions, and user presence changes.
+
+
+## Define AsyncAPI Version, API Information, and Server
```

Comment:
```suggestion
## Define AsyncAPI version, API information, and server
```

We observe sentence capitalization for headers, let's make sure to fix them all :)